src/app/events/events-governator/voteCreate.ts

```diff
@@ -0,0 +1,41 @@
+import { DiscordEvent } from '../../types/discord/DiscordEvent';
+import { createLogger } from '../../utils/logger';
+import { Client, TextChannel } from 'discord.js';
+import { VoteCreateEvent } from '../../types/governator-events/GovernatorEventTypes';
+import Api from '../../utils/api';
+import { ClientConfigDiscordDto } from '../../types/governator-api/GovernatorApiTypes';
+import { embedToEmbedBuilder, updateMessageEmbed, updateCountField } from '../../service/vote/Vote';
+
+export default class implements DiscordEvent {
+
+    name = 'VOTE_CREATE';
```

Comment:
Is this something the SDK would contain?

Reply:
no, that is to handle new SSE event. When vote from FE comes in it will update counter on the bot